packages/plugins/plugin-navtree/src/NavTreePlugin.tsx

```diff
@@ -130,18 +131,21 @@ export const NavTreePlugin = (): PluginDefinition<NavTreePluginProvides> => {
   return {
     meta,
     ready: async (plugins) => {
+      state.prop({ key: 'open', type: LocalStorageStore.json<string[]>() });
```

Comment:
this needs to be first so that state is restored from storage before anything acts on it